RecoEgamma/EgammaElectronProducers/python/gedGsfElectrons_cfi.py

```diff
@@ -19,9 +19,27 @@
                                          "gedelectron_EBUncertainty_offline_v1",
                                          "gedelectron_EEUncertainty_offline_v1"],
     combinationRegressionWeightLabels = ["gedelectron_p4combination_offline"],
+
+    #Activate the evaluation of Egamma PFID DNN
+    EleDNNPFid= dict(
+        enabled = True,
+        inputTensorName = "FirstLayer_input",
+        outputTensorName = "sequential/FinalLayer/Softmax",
+        modelsFiles = [
+            "RecoEgamma/ElectronIdentification/data/Ele_PFID_dnn/lowpT/lowpT_modelDNN.pb",
+            "RecoEgamma/ElectronIdentification/data/Ele_PFID_dnn/highpTEB/highpTEB_modelDNN.pb",
+            "RecoEgamma/ElectronIdentification/data/Ele_PFID_dnn/highpTEE/highpTEE_modelDNN.pb"
+        ],
+        scalersFiles = [
+            "RecoEgamma/ElectronIdentification/data/Ele_PFID_dnn/lowpT/lowpT_scaler.txt",
+            "RecoEgamma/ElectronIdentification/data/Ele_PFID_dnn/highpTEB/highpTEB_scaler.txt",
+            "RecoEgamma/ElectronIdentification/data/Ele_PFID_dnn/highpTEE/highpTEE_scaler.txt"
+        ]
+    )    
```

Comment:
it should not be necessary to duplicate here, as it was already specified in the fillDescriptions.

Reply:
In my opinion it would be good to specify here again at least the models and scalers files.  As soon as the final version of the models will be available we will refer to them here with more specific names, different from the default ones.